docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.9'
+
+services:
+    entropay-users-auth:
+      build: .
+      ports:
+          - "8080:8080"
```

Comment:
can we define in a wiki or similar which are going to be the default ports for the microservices? cc @agustinb4

Reply:
Sure, let's discuss it along with @agustinb4

Reply:
@arael82 I already update the readme file with the default port: 8000